src/modules/UI/scenes/CreateWallet/CreateWalletConnector.js

```diff
@@ -0,0 +1,3 @@
+import CreateWallet from './CreateWallet.ui.js'
+
+export default CreateWallet
```

Comment:
Do not use default exports. Per our convo in engineering meeting, use explicitly named exports in all new code